h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -1357,43 +1357,53 @@ def show_test_results(test_name, curr_test_val, new_test_val):
         return 0
 
 
-def assert_H2OTwoDimTable_equal(table1, table2, col_header_list, tolerance=1e-6, check_sign=False):
+def assert_H2OTwoDimTable_equal(table1, table2, col_header_list, tolerance=1e-6, check_sign=False, check_all=True,
```

Comment:
Don't write methods with two boolean arguments. Write do different ones (or four, if needed... is it needed?)